e2e/cypress/integration/system_console/reporting/site_statistics_spec.js

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Group: @system_console
+
+import * as TIMEOUTS from '../../../fixtures/timeouts';
+
+describe('System Console > Site Statistics', () => {
+    before(() => {
+        // # Visit site statistics page
+        cy.visit('/admin_console/reporting/system_analytics').wait(TIMEOUTS.TWO_SEC);
+    });
+
+    it('MM-T904 Site Statistics displays expected content categories', () => {
+        // * Check that the header has loaded correctly and contains the expected text.
+        cy.get('.admin-console__header span').should('be.visible').should('contain', 'System Statistics');
+
+        // * Check that the rows for the table were generated.
+        cy.get('.admin-console__content .row').should('have.length', 4);
```

Comment:
You are probably on a licensed server. The test is currently *calibrated* for team edition.
I think we can either remove the order check so that it works on both or make sure it's executed in one version only.

Reply:
Ah, I see. I suggests to have a separate test for enterprise and team edition. I'll update the TM4J test case later. We normally run with license in CI but we handle test cases for team edition as well by removing the license prior test using `cy.apiDeleteLicense()`. Don't worry about re-uploading the license. There's `uploadLicenseIfNotExist` that handles it whenever needed.